test/NonSiloTests/Orleans.NonSiloTests.csproj

```diff
@@ -74,6 +98,18 @@
       <Link>Version.txt</Link>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <Content Include="..\..\src\SDK\OrleansConfiguration.xml">
+      <Link>OrleansConfiguration.xml</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="..\TesterInternal\Data\TestDb.mdf">
+      <Link>Data\TestDb.mdf</Link>
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </Content>
+    <Content Include="..\TesterInternal\Data\TestDb_log.ldf">
+      <Link>Data\TestDb_log.ldf</Link>
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </Content>
```

Comment:
Are these needed for something? I remember asking around and nobody knew whay it was needed or whether we shold just kill it